flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/CheckpointFailureManagerTest.java

```diff
@@ -96,7 +96,7 @@ public void testTotalCountValue() {
         }
 
         // CHECKPOINT_DECLINED, CHECKPOINT_EXPIRED and CHECKPOINT_ASYNC_EXCEPTION
-        assertEquals(3, callback.getInvokeCounter());
+        assertEquals(5, callback.getInvokeCounter());
```

Comment:
Inconsistency between number and comment.

Reply:
I have fixed this too.